src/main/kotlin/racing/Car.kt

```diff
@@ -0,0 +1,42 @@
+package racing
+
+class Car private constructor(
+    sequence: Int,
+    position: Int,
+    private var generator: NumberGenerator,
```

Comment:
자동차의 움직임을 위한 인터페이스를 자동차를 생성할 때 잘 넣어주셨습니다 👍👍

```suggestion
    private var generator: NumberGenerator = RandomNumberGenerator(), 
```

defaultValue를 통해서 기본적으로 넣어줄 수도 있어 보이는데요! 이건 취향 차이라고 생각해요~😊